app/routes/lending/LendableObjectsList.tsx

```diff
@@ -0,0 +1,217 @@
+import { Button, Card } from '@webkom/lego-bricks';
+import { usePreparedEffect } from '@webkom/react-prepare';
+import qs from 'qs';
+import { Helmet } from 'react-helmet-async';
+import { Link, useHistory, useLocation } from 'react-router-dom';
+import { Content } from 'app/components/Content';
+import TextInput from 'app/components/Form/TextInput';
+import { Image } from 'app/components/Image';
+import NavigationTab, { NavigationLink } from 'app/components/NavigationTab';
+import type { ListLendableObject } from 'app/store/models/LendableObject';
+import styles from './LendableObjectsList.css';
+import moment from 'moment';
+import { useState } from 'react';
+import { LendingRequest, status } from './components/LendingRequest';
+
+const LendableObject = ({
+  lendableObject,
+}: {
+  lendableObject: ListLendableObject;
+}) => {
+  return (
+    <Link to={`/lending/${lendableObject.id}`}>
+      <Card isHoverable hideOverflow className={styles.lendableObjectCard}>
+        <Image
+          className={styles.lendableObjectImage}
+          src={lendableObject.image}
+          alt={`${lendableObject.title}`}
+        />
+        <div className={styles.lendableObjectFooter}>
+          <h2>{lendableObject.title}</h2>
+        </div>
+      </Card>
+    </Link>
+  );
+};
+
+type Query = {
+  title?: string;
+};
+
+const parseQuery = (search: string): Query => {
+  const { title } = qs.parse(search, {
+    ignoreQueryPrefix: true,
+  });
+
+  return {
+    title: typeof title === 'string' ? title : undefined,
+  };
+};
+
+export const LendableObjectsList = () => {
+  const location = useLocation();
+  const query = parseQuery(location.search);
+  const history = useHistory();
```

Comment:
`useHistory` is not part of `react-router` v6. Use `useNavigation` instead

Reply:
(but you'll have to rebase first of course)

Reply:
changed this to use useSearchParams, felt more accurate